sdk/core/azure-core-amqp/samples/eventhub_async_writer_sample/eventhub_async_writer_sample.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-Licence-Identifier: MIT
+
+#include <azure/core/amqp/connection.hpp>
+#include <azure/core/amqp/message_sender.hpp>
+#include <chrono>
+#include <iostream>
+#include <limits>
+#include <string>
+
+#define EH_CONNECTION_STRING "<<<Replace with the connection string from your eventhubs instance>>>"
+
+int main()
+{
+  auto credentials{
+      std::make_shared<Azure::Core::Amqp::_internal::SaslPlainConnectionStringCredential>(
+          EH_CONNECTION_STRING)};
+  std::string targetUrl
+      = "amqps://" + credentials->GetHostName() + "/" + credentials->GetEntityPath();
+  Azure::Core::Amqp::_internal::ConnectionOptions connectOptions;
+  connectOptions.ContainerId = "some";
+  connectOptions.HostName = credentials->GetHostName();
+  connectOptions.SaslCredentials = credentials;
+  //  connectOptions.EnableTrace = true;
```

Comment:
Why is this commented out?

Reply:
Because it introduces noise to the sample.